### PR TITLE
Escrow-JS Example

### PR DESCRIPTION
**Pull request by idea404**
This PR adds the `escrow-js` example to the docs examples. 

Contract found: https://github.com/near-examples/escrow-js

### CODE REVIEW

File: website/sidebars.json [hunk 1]
```diff
@@ -336,6 +336,13 @@
         "tutorials/examples/update-contract-migrate-state"
       ]
     },
+    "tutorials/examples/hello-near",
+    "tutorials/examples/count-near",
+    "tutorials/examples/guest-book",
+    "tutorials/examples/donation",
+    "tutorials/examples/escrow",
+    "tutorials/examples/xcc",
+    "tutorials/examples/coin-flip",
```
Comment (bucanero) [inline website/sidebars.json:345]: these examples are already listed under the "Basic Examples" sidebar group.

I'd add the escrow example there, but if you think it should remain outside the group, ping Guillermo just to be sure he's ok with it.

File: docs/3.tutorials/examples/escrow.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+---
+id: escrow
+title: Escrow Contract
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import {CodeTabs, Language, Github} from "@site/components/codetabs"
+
+This example provides a simple example for an escrow contract, with a timeout callable function that concludes the transaction if the buyer has not provided an approval within a given timeframe.
```
Comment (bucanero) [inline docs/3.tutorials/examples/escrow.md:9]:
```suggestion
This example provides a simple implementation of an escrow contract, with a timeout callable function that concludes the transaction if the buyer has not provided approval within a given timeframe. 
```

File: docs/3.tutorials/examples/escrow.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+The code also includes an example of how contracts can pass data to each other through cross contract calls, and how the caller contract can handle the result.
+
+:::info
+This example does not have a frontend
+:::
+
+---
+
+## Starting with the Project
+You have two options to start using the project. The first and recommended is to open the project through Gitpod, which will open a web-based interactive environment with all dependencies installed. The second option is to clone the repository locally, for which you will need to install all the [Prerequisites](../../2.develop/prerequisites.md).
```
Comment (bucanero) [inline docs/3.tutorials/examples/escrow.md:20]:
```suggestion
You have two options to start using the project:
- The first and recommended is to open the project through Gitpod, which will open a web-based interactive environment with all dependencies installed.
- The second option is to clone the repository locally, for which you will need to install all the [Prerequisites](../../2.develop/prerequisites.md).
```

File: docs/3.tutorials/examples/escrow.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+#### 1. Install Dependencies
+To install dependencies, run:
+
+```bash
+npm i
+```
+
+#### 1. Build the Contracts
+You can build the contract (creating the `.wasm` file that is ultimately deployed to the network) by running:
```
Comment (bucanero) [inline docs/3.tutorials/examples/escrow.md:49]:
```suggestion
You can build the contracts (creating the `.wasm` files that will be deployed to the network) by running:
```

File: docs/3.tutorials/examples/escrow.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+            url="https://github.com/near-examples/escrow-js/blob/master/contracts/escrow.js"
+            start="41" end="119" />
+  </Language>
+</CodeTabs>
+
+---
+
+## Testing
+
+When writing smart contracts it is very important to test all methods exhaustively. In this
```
Comment (bucanero) [inline docs/3.tutorials/examples/escrow.md:133]:
```suggestion
When writing smart contracts it is critical to test all methods exhaustively. In this
```

File: website/sidebars.json (hunk 1 above)
Comment (idea404) [reply website/sidebars.json]: Thanks for pointing this out! I must have missed it while merging master. I've added `escrow` to the "Advanced Examples" area

**Review by bucanero — APPROVED** [review]

**Pull request merged**